WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -556,6 +568,42 @@ - (void)configureMediaUpload
     self.mediaProgressView = [[UIProgressView alloc] initWithProgressViewStyle:UIProgressViewStyleBar];
 }
 
+- (void)createMediaThumbnailCallback:(NSURL *)thumbnailURL mediaID:(NSString *)mediaID isImage:(BOOL)isImage
```

Comment:
I would probably remove `callback` from the name of this method. Maybe something like `handleThumbnailURL:mediaID:isImage:`?